intelmq/lib/message.py

```diff
@@ -221,18 +221,27 @@ def __get_type_config(self, key):
         return class_name
 
     def __hash__(self):
+        return int(self.hash(), 16)
```

Comment:
The `hash`  method returns a string, but `__hash__` has to return an int. This conversion from string to int is the only part left in `__hash__` from its original version.